frontend/pages/index.js

```diff
@@ -381,111 +444,185 @@ const Homepage = () => {
                         textColor="blue.900"
                         alignSelf={["center", "center", "initial"]}
                         href="https://docs.google.com/document/d/1mjfF8SgRrAZvtCVVxB2qNSUcbbmrH6dTEYSMfHKdEgc/preview"
+                        isExternal
                       >
-                        Explore case studies
+                        Explore the Use Cases
                       </RouteButton>
-                    </VStack>
-                  </GridItem>
-                  <GridItem>
-                    <Flex>
-                      <chakra.span
-                        fontSize={["md", "2xl", "3xl", "3xl", "3xl", "4xl"]}
-                        display="inline-block"
-                        color="blue.200"
-                      >
-                        Lootboxes, crafting, deck builder, you name it! Whatever
-                        on-chain mechanics you want incorporated in your
-                        project, contact us to help you launch it. It is fast
-                        and secure. Or explore the features to know more.
-                      </chakra.span>
-                    </Flex>
+                    </Center>
                   </GridItem>
                 </Grid>
-                <Divider mt={12} mb={12} />
                 <SimpleGrid
                   columns={[1, 2, 2, 4, null, 4]}
                   justifyContent="center"
                   w="100%"
                   placeContent={"space-between"}
+                  mx={[0, -2, -4]}
                 >
                   <Feature
                     image={assets["cryptoTraders"]}
                     altText="mined transactions"
-                    heading="ON-CHAIN MECHANICS"
+                    heading="Lootboxes"
                   />
                   <Feature
                     image={assets["NFT"]}
                     altText="mined transactions"
-                    heading="LOYALTY PROGRAMS"
+                    heading="Crafting Recipes"
                   />
                   <Feature
-                    image={assets["lender"]}
+                    image={assets["DAO"]}
                     altText="mined transactions"
-                    heading="SECURE TRANSACTIONS"
+                    heading="Minigames"
                   />
                   <Feature
-                    image={assets["DAO"]}
+                    image={assets["lender"]}
                     altText="mined transactions"
-                    heading="CONTENT MANAGEMENT"
+                    heading="Airdrops"
                   />
                 </SimpleGrid>
               </GridItem>
               <GridItem
-                px="7%"
+                paddingLeft="15%"
```

Comment:
Paddings on left and right on grid item should be equal. Use ` px={["7%", null, "12%", "15%"]}` or `px="7%"` and solve any other required paddings by using css aligning/justifying, if that's not enough - You can add any additional paddings you need in child components of GridItem

Reply:
<img width="725" alt="image" src="https://user-images.githubusercontent.com/61459744/167010820-52a81323-2339-4187-9bac-22e5a361ea8e.png">